data-raw/standard-format-data-prep/flow_data_prep_files/flow_data_prep.Rmd

```diff
@@ -512,14 +512,64 @@ filter(oroville_daily_flows, flow_cfs == 0)
 filter(oroville_daily_flows, is.na(flow_cfs))
 min(oroville_daily_flows$flow_cfs)
 ```
+
+### Lower Feather site
+
+```{r}
+# Find CDEC gage nearby the Lower Feather RST site
+# Use this code to see what data are available
+cdec_datasets("FSB") 
+# For reference - this mapping tool shows CDEC sites: https://cdec.water.ca.gov/webgis/?appid=cdecstation
+
+# Use this code to pull the data from CDEC. 
+boyds_landing_CDEC <- cdec_query(station = "FSB", dur_code = "H", 
+                                 sensor_num = "20", start_date = "1996-01-01")
+
+# Format data
+boyds_landing_hourly_flows <- boyds_landing_CDEC %>%
+  mutate(date = as_date(datetime),
+         time = as_hms(datetime),
+         flow_cfs = parameter_value)
+  
+```
+
+
+#### QA/QC
+
+```{r}
+ggplot(boyds_landing_hourly_flows, aes(x = date, y = flow_cfs)) +
+  geom_line()
+# just checking to make sure there aren't any zeros
+boyds_landing_hourly_flows |> filter(flow_cfs == 0) |> nrow()
+boyds_landing_hourly_flows |> filter(is.na(flow_cfs)) |> nrow()
+boyds_landing_hourly_flows |> filter(flow_cfs < 0) |> nrow()
+boyds_landing_hourly_flows |> nrow()
+```
+
+#### Clean data
+
+```{r}
+boyds_landing_hourly_flows_clean <- boyds_landing_hourly_flows |> 
+  mutate(flow_cfs = ifelse(flow_cfs < 0, NA_real_, flow_cfs))
+
+boyds_landing_hourly_flows_clean_mean <- boyds_landing_hourly_flows_clean |> 
```

Comment:
You can combine these both (add mutate statement to second code chunk). Also looks like we are grouping by day so I would call these boyds_landing_mean_daily_flows

Reply:
Also looks like there is one NA date number that I would remove